cmd/autoheal/awx_job.go

```diff
@@ -102,5 +102,29 @@ func (h *Healer) launchAWXJob(connection *awx.Connection, template *awx.JobTempl
 		"Request to launch AWX job from template '%s' has been sent",
 		templateName,
 	)
+
+	glog.Infof(
+		"Job id: %v",
+		response.Job,
+	)
+
+	h.watchJobStatus(connection, response.Job)
```

Comment:
If I understand correctly we are checking the job status just once. I assume that you will evolve that in future versions of the pull request. We will probably need to remember (in memory) the jobs that we created periodically check their status. A possible way to do that is to create another work queue, post to it the jobs that need checking, remove the jobs that have been completed and re-post the ones that aren't completed.